config/Map.yaml

```diff
@@ -119,3 +119,7 @@ mappings:
     newAPI: "apiVersion: rbac.authorization.k8s.io/v1\nkind: RoleBindingList"
     deprecatedInVersion: "v1.17"
     removedInVersion: "v1.22"
+  - deprecatedAPI: "apiVersion: networking.k8s.io/v1beta1\nkind: Ingress"
+    newAPI: "apiVersion: networking.k8s.io/v1\nkind: Ingress"
+    deprecatedInVersion: "v1.19"
+    removedInVersion: "v1.22"
```

Comment:
Can you move this to where the other `Ingress` entry is in the file?

Reply:
Sure - I changed the position below the other `Ingress` entry.